ipam/ippool_manager.go

```diff
@@ -253,6 +343,55 @@ func (m *IPPoolManager) updateAddress(ctx context.Context,
 	return addresses, nil
 }
 
+func (m *IPPoolManager) K8sUpdateAddress(ctx context.Context,
```

Comment:
```suggestion
func (m *IPPoolManager) capiUpdateAddress(ctx context.Context,
```
Please add a doc string also. I know the other one doesn't have it either, but let's anyway try to improve it as we go along :slightly_smiling_face: